### DISCUSSION TITLE
Anyone running the new Pytorch release?

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#2292] [General]
Pytorch 2.1 just came out a few days ago.

I'm still running with 2.0, CUDA 11.8 toolkit and cudnn 8.9.4 for 11.x on my Nvidia 4090 with 531.79 drivers. It performs fast but not at the top speeds posted by others.

Anyone else running with any of the newer stuff? What's your setup look like and how is it performing?

**Comment by vladmandic** [comment]
i'm always on the latest - i'd never suggest it to everyone, but i need to in order to make sure sdnext is compatibile with whatever is coming.
is there a performance benefit? not really.
torch 2.2.0.dev, cuda 12.1, nvidia 537.42
